tutorials/users/ide/version-control.md

```diff
@@ -1 +1,612 @@
+
+```
+git init
+```
+
+![Terminal Git Init](images/version-control/terminal-git-init.png)
+
+The output shown in the screenshot above confirms that a new empty Git repository was initialized.
+
+### Adding a Remote Repository
+
+```{note}
+For this step, you will need an online (remote) Git repository. If you don't already have one, and if you unfamiliar with
+creating one, then go to the [](github) section and follow the steps there.
+```
+
+You can optionally add a remote repository. Adding a remote repository allows you to store your work online at GitHub and access it anywhere. You can add a remote
+repository for the local repository you created in the [previous step](create_local_repo) by clicking on `Git` in the menu bar and then selecting `Add Remote Repository`. Enter the URL of the [remote repository on GitHub](github_remote_repo) into the `Add Remote Repository` dialog box.  Click `OK`. If you 
+prefer to use the terminal, then navigate to your Git repo folder and issue the following command:
+
+```
+git remote add origin url-of-your-git-repository
+```
+
+### Cloning an Existing Repository
+
+Cloning a repository is another way of adding a remote repository. Cloning creates a copy of an existing remote repository in your local folder.
+
+Find a repository online you want to clone and copy the URL of the repository.
+
+Create a new folder or navigate to the folder you want to keep this repository in.
+
+Click the `Git Clone` button in the breadcrumb menu
+
+![Git Clone](images/version-control/git-clone-breadcrumb.png)
+
+```{note}
+This button is only active when you are in a folder that does not contain a git repository.
+``` 
+
+Paste the repository URL into the dialog box. Click `Clone`. The online repository's files will be copied into the local folder which will
+then be initialized as a local Git repository and the remote repository will automatically be configured to be the one from which you cloned.
+
+
+## Using Git with the Graphical User Interface
+By default when you create a new repository you will be using a default branch, which is usually named **main**.
+This is the main branch where the initial work files will be pushed to and also where the final work version will be.
+
+It is good practice to create work branches from the main branch where all the changes to your work will be done.
+Once the changes are complete they can be pushed and merged with the main branch
+
+### Making Your First Commit
+
+Start by creating a file.  In this example a Jupyter Notebook file is created by clicking one of the Jupyter Notebook buttons in the launcher.
+
+![Jupyter Notebook Launcher Buttons](images/version-control/jupyter-notebook-buttons.png)
+
+When the file is created or altered it will be seen under the `Untracked` section in the Git sidebar.
+
+![Git Untracked](images/version-control/git-untracked.png)
+
+### Staging a File
+Staging a file means getting the file ready to be committed and pushed to the remote repository.  
```

Comment:
I don't think staging itself has anything to do with pushing to a remote repository.

Reply:
Staging has to to with committing, and committing has to do with pushing to a remote repository.  I guess it depends on how you interpret the sentence.

I'll change it to make it less ambiguous.

Reply:
Could be: 

> Staging a file means adding it to the list of files that will be included the next time you make a commit.

Then drop the rest since it is covered in the "Push a commit" section below

Reply:
What I was thinking here is that since we are saying in our tutorial that adding an external repo is an optional step, this might come off as saying that committing is tied to an pushing (and the need for an external repo).

Reply:
> Staging a file means adding it to the list of files that will be included the next time you make a commit. 

That's a good one.  I'll change it to this.